extensions/wikia/Search/classes/QueryService/Select/AbstractSelect.php

```diff
@@ -197,92 +222,100 @@ public function searchAsApi( $fields = null, $metadata = false, $keyField = null
 		} else {
 			$response = $resultSet->toArray( null, $keyField );
 		}
+
 		return $response;
 	}
-	
+
 	/**
-	 * Retrieves an existing match, or forces the child class to retrieve a match. 
-	 * @return \Wikia\Search\Match\Article|\Wikia\Search\Match\Wiki|\Wikia\Search\false|boolean
+	 * Retrieves an existing match, or forces the child class to retrieve a match.
+	 *
+	 * @return \Wikia\Search\Match\Article|\Wikia\Search\Match\Wiki|boolean
 	 */
 	public function getMatch() {
 		$config = $this->getConfig();
 		if ( $config->hasMatch() ) {
 			return $config->getMatch();
 		}
+
 		return $this->extractMatch();
 	}
-	
+
 	/**
-	 * This hook should be overidden by children to access the appropriate kind of match.
-	 * @return NULL
+	 * This hook should be overridden by children to access the appropriate kind of match.
+	 *
+	 * @return null
 	 */
 	protected function extractMatch() {
 		return null;
 	}
-	
+
 	/**
 	 * Creates an instance of Solarium_Query_Select configured by searchconfig.
+	 *
 	 * @return \Solarium_Query_Select
 	 */
 	protected function getSelectQuery() {
 		$query = $this->getClient()->createSelect();
 		$query->setDocumentClass( '\Wikia\Search\Result' );
-		$this->registerQueryParams( $query )
-		     ->registerComponents( $query );
+		$this->registerQueryParams( $query )->registerComponents( $query );
+
 		return $query->setQuery( $this->getQuery() );
 	}
-	
+
 	/**
 	 * This is abstract because the dismax and lucene query services behave differently here.
+	 *
 	 * @return string
 	 */
 	abstract protected function getQuery();
-	
+
 	/**
 	 * Allows us to configure components in child instances.
+	 *
 	 * @param \Solarium_Query_Select $query
+	 *
 	 * @return \Wikia\Search\QueryService\Select\AbstractSelect
 	 */
 	protected function registerComponents( Solarium_Query_Select $query ) {
 		return $this;
 	}
-	
+
 	/**
 	 * Registers meta-parameters for the query
+	 *
 	 * @param Solarium_Query_Select $query
+	 *
 	 * @return \Wikia\Search\QueryService\Select\AbstractSelect
 	 */
 	protected function registerQueryParams( Solarium_Query_Select $query ) {
 		$config = $this->getConfig();
 		$sort = $config->getSort();
-		$query->addFields      ( $this->getRequestedFields() )
-		      ->removeField    ('*')
-		      ->setStart       ( $config->getStart() )
-		      ->setRows        ( $config->getLength() )
-		      ->addSort        ( $sort[0], $sort[1] )
-		      ->addParam       ( 'timeAllowed', $this->timeAllowed )
-		;
+		$query->addFields( $this->getRequestedFields() )->removeField( '*' )->setStart( $config->getStart() )->setRows(
+			$config->getLength()
+		)->addSort( $sort[0], $sort[1] )->addParam( 'timeAllowed', $this->timeAllowed );
+
 		return $this;
 	}
-	
+
 	/**
 	 * For now, this is the union of the default requested fields (usually required for minimum functionality),
 	 * and any fields specifically added by the client code.
+	 *
 	 * @todo Add support for _removing_ requested fields?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)